.github/workflows/verify-build.yml

```diff
@@ -0,0 +1,69 @@
+# This workflow will perform a test whenever there
+# is some change in code done to ensure that the changes
+# are not buggy and we are getting the desired output.
+name: Test Build
+on: [push, pull_request]
+jobs:
+  build:
+    name: Build image using Buildah
+    runs-on: ubuntu-latest
+    steps:
+      # Fetch name of the Forked Repository
+      - name: Fetch PR head repo
+        if: github.event_name == 'pull_request'
+        id: head_repo_name
+        run: |
+          HEAD_REPO_NAME=$(jq -r '.pull_request.head.repo.full_name' "$GITHUB_EVENT_PATH")
+          echo "PR head repo: $HEAD_REPO_NAME"
+          echo "::set-output name=repo::$HEAD_REPO_NAME"
+      # Extract branch name
+      - name: Extract branch name
```

Comment:
Please put a blank line between steps

Reply:
Sure :+1: 
Actually I did but yaml linter removed that :stuck_out_tongue: